airflow/providers/amazon/aws/operators/emr_add_steps.py

```diff
@@ -52,34 +53,36 @@ class EmrAddStepsOperator(BaseOperator):
     @apply_defaults
     def __init__(
         self,
-        *,
-        job_flow_id=None,
-        job_flow_name=None,
-        cluster_states=None,
-        aws_conn_id='aws_default',
-        steps=None,
+        job_flow_id: Optional[str] = None,
+        job_flow_name: Optional[str] = None,
+        cluster_states: Optional[List[str]] = None,
+        aws_conn_id: str = 'aws_default',
+        steps: Optional[Union[List[dict], str]] = None,
+        *args,
```

Comment:
```suggestion

```
As we cannot pass `args` right?

Reply:
Yes, we shouldn't allow to pass `*args` to operators init. That's why we have the `*`.